module/src/components/button/button.component.tsx

```diff
@@ -30,46 +30,58 @@ export interface IButtonProps
 }
 
 /** Renders an HTML button element with some useful additions */
-export const Button: React.FC<IButtonProps> = ({
-  className,
-  validationErrorMessages,
-  errorIcon,
-  pending,
-  disabled,
-  error,
-  leftIcon,
-  rightIcon,
-  children,
-  statusPosition,
-  hideIconOnStatus,
-  ...nativeProps
-}) => {
-  const shouldShowErrorIcon = !!validationErrorMessages?.length || error;
```

Comment:
is `error` already a bool or does it need to be bool cast to ensure that the final const is a bool?
```
const shouldShowErrorIcon = !!validationErrorMessages?.length || !!error;
```
If it's already a bool then ignore me.

Reply:
It is indeed already a bool